src/main/java/bio/terra/service/dataset/DatasetRequestValidator.java

```diff
@@ -184,12 +184,17 @@ private void validateTable(TableModel table, Errors errors, SchemaValidationCont
     String tableName = table.getName();
     List<ColumnModel> columns = table.getColumns();
     List<String> primaryKeyList = table.getPrimaryKey();
+    List<String> columnNames = new ArrayList<>();
+    if (columns == null || columns.isEmpty()) {
+      errors.rejectValue(
+          "schema", "IncompleteSchemaDefinition", "Each table must at least one column");
```

Comment:
Should this read `"Each table must contain at least one column"`?

Reply:
yes, I definitely skipped a word in there...